### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by adrien-zenchef** [Roboroads/laravel-tinker#247]
### Short bug description

The plugin does not resolve any vendor import.

### Extended bug description

Since new PHPStorm + plugin update nothing is working anymore :(

### Your project is not the problem

- [X] I tried using a fresh laravel install - I'm fairly certain it has nothing to do with my project.

### How to reproduce?

```php
use Illuminate\Support\Facades\Request;

$request = Request::create(
    '/api/v1/endpoint',
    'GET',
    ['param' => 'value']
);
```

### Expected behavior

Show request object

### Screenshots

![image](https://user-images.githubusercontent.com/106613033/231705740-f93008cb-a898-4816-976e-a75b6bc5b610.png)

### IDE & Version

PHPStorm 2023.1

### PHP Version

8.0.28

### OS and version

Mac OS (M1 PRO)

### Laravel Tinker plugin version

2.5.1

### Interpreter type

Local

### Laravel Version

7.30.6

### Tinker version

2.8.1

### Psysh version

0.11.13

**Comment by Roboroads** [comment]
This is strange. Is the output displayed in the tinker toolwindow? How are you executing the code?

It feels like the tinker plugin is not handling your code here, but something else is.

**Comment by adrien-zenchef** [comment]
> This is strange. Is the output displayed in the tinker toolwindow? How are you executing the code?
> 
> It feels like the tinker plugin is not handling your code here, but something else is.

Yes the output is displayed in the tinker toolwindow. I click on the green right arrow.
But I think it's not using my local php interpreter (like before) but a remote from 3v4l.org

Output https://3v4l.org/jjukv

Yes exactly, I have the same impression....

**Comment by adrien-zenchef** [comment]
When I ran something like

```php
<?php 
echo 'hello world';
```

it's working, but as soon as I import something from my project it doesn't work anymore. It just seems that the code is running in remote and not in local as I would like

**Comment by Roboroads** [comment]
So, the issue is, that the interpreter you are using should also have all the files - like a synched folder or something. Check your PHP settings to see which interpreter you are using currently, the plugin just uses that.

**Comment by adrien-zenchef** [comment]
My interpreter is well configured, I use it for tests and with the tools line command of the IDE for example. I don't understand why instead of using my interpreter it uses a remote interpreter... 
![image](https://user-images.githubusercontent.com/106613033/232410854-57ec4c8b-6a54-4fad-87c4-88c9f759272f.png)

**Comment by Roboroads** [comment]
Note that's just the name of the interpreter. Check under the three dots next to the interpreter if it';s actually the local interptreter and that you didn't change the name on accident.

**Comment by adrien-zenchef** [comment]
The PHP interpreter is dected, as I said I use it for tests/command line tools shortcut. On `...` it's well detected

**Comment by Roboroads** [comment]
I don't know how I can help you any more than this. As per https://github.com/Roboroads/laravel-tinker/blob/main/src/main/kotlin/nl/deschepers/laraveltinker/util/PhpArtisanTinkerUtil.kt#L48 it's just picking the prject interpreter.

Maybe reinstall Tinker?

**Comment by Roboroads** [comment]
I forgot to mention 1 thing. In the settings of the laravel plugin, you can also change the path of your laravel folder within the interpreter. You can find those in your settings as well.

**Comment by Roboroads** [comment]
I've added all the info into #250

**Comment by adrien-zenchef** [comment]
Unfortunately I have tried everything but nothing...
On my Windows everything works perfectly, on Mac nothing works. Thanks for your help, we can close the ticket I guess

**Comment by Roboroads** [comment]
Hi! I just got this 3v4l.org thing in my PHPStorm as well. But I still got tinker to keep working. Make sure you press the right button: 

![image](https://user-images.githubusercontent.com/4398301/233642441-94b79aea-1a0b-4f14-b1b9-6a861a4eb483.png)

**Comment by Roboroads** [comment]
@Jelle-SamsonIT I did not change anything. It's JB that added this new 3vl4 interpreter stuff.

If I've spent countless hours on figuring out to *just* have the syntax highlighting and none of the other stuff that comes with PHP for the tinker consoles. However the PHP plugin (from JB) is not documented **at all** so it's extremely hard to figure things out.
